files/disable_inactive_freeipa_users.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/bash
+
+# Usage:
+#   disable_inactive_freeipa_users.sh [disable_time]
+#
+# A user is considered inactive if he or she has not logged since
```

Comment:
Use more inclusive wording and add a missing word:
```suggestion
# A user is considered inactive if they have not logged in since
```

Reply:
"They" is a plural pronoun.  How about this:
```suggestion
# Users are considered inactive if they have not logged in since
```

Reply:
See [8d971f1](https://github.com/cisagov/ansible-role-freeipa-server/pull/73/commits/8d971f1de83ce7394cb916e1fc1e2ce418671208) for the missing word.

Reply:
> "They" is a plural pronoun. How about this:

That works for me. 👍

Reply:
Please see commit [43c6240](https://github.com/cisagov/ansible-role-freeipa-server/pull/73/commits/43c6240629a59d716428f7602f86189967251f81).